requirements-dev.txt

```diff
@@ -0,0 +1,11 @@
+git+https://github.com/asdf-format/asdf
+git+https://github.com/asdf-format/asdf-standard
+git+https://github.com/astropy/astropy
+git+https://github.com/astropy/asdf-astropy
+git+https://github.com/spacetelescope/crds
+git+https://github.com/spacetelescope/romancal
+git+https://github.com/spacetelescope/roman_datamodels
+git+https://github.com/spacetelescope/romanad
+git+https://github.com/spacetelescope/stpipe
+git+https://github.com/spacetelescope/stcal
+git+https://github.com/spacetelescope/stdatamodels
```

Comment:
I would include also
```
git+https://github.com/asdf-format/asdf-transform-schemas
git+https://github.com/asdf-format/asdf-wcs-schemas
git+https://github.com/spacetelescope/gwcs
```
Do we need `stdatamodels`? It's a dependency of `stpipe`, right? And is not used directly by roman packages.

Reply:
I've added those three packages.
You are correct stdatamodels is only used by stpipe. The roman calibration does not use stdatamodels except for dq_init_tests,
from stdatamodels.validate import ValidationWarning
to filter warnings on the ref files.

Reply:
Learning from jwst, if asdf-standard is here, pip will not be able to resolve the version conflicts. So remove `asdf-standard` and I think it's ready to go.

Reply:
I removed asdf-standard from the dependency list.